ament_cmake_auto/cmake/ament_auto_find_build_dependencies.cmake

```diff
@@ -15,6 +15,11 @@
 #
 # Invoke find_package() for all build and buildtool dependencies.
 #
+# :param REQUIRED_PACKAGES: an optional list of package names that are known
```

Comment:
I would suggest to name the keyword `REQUIRED` to be shorter and more similar for CMake.